app/webpack.config.prod.js

```diff
@@ -0,0 +1,29 @@
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+
+module.exports = {
+    mode: 'production',
+    output: {
+        filename: '[name].[contenthash].js',
+    },
+    optimization: {
+        minimizer: [
+            `...`,
```

Comment:
what is it for?

Reply:
```
      // For webpack@5 you can use the `...` syntax to extend existing minimizers (i.e. `terser-webpack-plugin`), uncomment the next line
      // `...`

```
https://webpack.js.org/plugins/html-minimizer-webpack-plugin/

![image](https://user-images.githubusercontent.com/22503765/108234810-5d90a080-716f-11eb-8b18-56ba5a06581e.png)